include/zephyr/drivers/usb_c/usbc_pd.h

```diff
@@ -97,6 +97,104 @@ extern "C" {
  */
 #define PD_T_SAFE_5V_MAX_MS  275
 
+/**
+ * @brief Time to wait for TCPC to complete transmit
+ */
+#define PD_T_TX_TIMEOUT_MS 100
+
+/**
+ * @brief Minimum time a Hard Reset must complete.
+ *	  See Table 6-68 Time Values
+ */
+#define PD_T_HARD_RESET_COMPLETE_MIN_MS 4
+
+/**
+ * @brief Maximum time a Hard Reset must complete.
+ *	  See Table 6-68 Time Values
+ */
+#define PD_T_HARD_RESET_COMPLETE_MAX_MS 5
+
+/**
+ * @brief Minimum time a response must be sent from a Port Partner
+ *        See Table 6-68 Time Values
+ */
+#define PD_T_SENDER_RESPONSE_MIN_MS 24
```

Comment:
USB Power Delivery Specification Revision 3.1, Version 1.5, Table 6-68 Time Values, tSenderResponse contains values min 26, nom 29, max 32 ms.

Is there a difference between USB Power Delivery Specification Revision 3.0 Version 2.0 (as noted in the usbc_pd.h header) and USB Power Delivery Specification Revision 3.1, Version 1.5 (the version currently available for download at usb.org Document Library) or are the values entered incorrectly here?

Reply:
Yes. It seems tSenderResponse changed in Revision 3.1, Version 1.4.
**Rev 3.1, Version 1.3**: 24 min, 27 nom, 30 max
**Rev 3.1, Version 1.4**: 26 min, 29 nom, 32 max